datadog/__init__.py

```diff
@@ -78,12 +78,13 @@ def initialize(
                                      (default: True).
     :type statsd_disable_buffering: boolean
 
-    :param statsd_disable_aggregator: Enable/disable statsd client aggregation support
+    :param statsd_disable_aggregation: Enable/disable statsd client aggregation support
```

Comment:
What the reason for changing the name? Just curious as if anyone using this feature today will have to update to using name now.

Reply:
I asked him to change this (and the other setting `disable_aggregating` to be consistent with the rest of our naming. (Since this feature is not documented yet, I expect it to be ok. - Would like to document this for the next release)